### PR TITLE
Added ERC721 metadata methods

### PR DESCRIPTION
**Pull request by DancingAxolotl**

### CODE REVIEW

File: Sources/UnstoppableDomainsResolution/Helpers/TokenUriMetadata.swift [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+public struct TokenUriMetadata : Codable {
+    let name: String
+    let description: String
+    let externalURL: String
```
Comment (rommex) [inline Sources/UnstoppableDomainsResolution/Helpers/TokenUriMetadata.swift:6]: `externalUrl`

**Review by rommex — APPROVED** [review]

File: Sources/UnstoppableDomainsResolution/Helpers/TokenUriMetadata.swift [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    let value: TokenUriMetadataValue
+
+    enum CodingKeys: String, CodingKey {
+        case displayType = "display_type"
+        case traitType = "trait_type"
+        case value
+    }
+}
+
+struct TokenUriMetadataValue: Codable {
```
Comment (DeRain) [inline Sources/UnstoppableDomainsResolution/Helpers/TokenUriMetadata.swift:42]: Let's add a few simple tests for this helper

**Review by DeRain — CHANGES_REQUESTED** [review]

**Review by merenkoff — APPROVED** [review]
LGTM if @DeRain  will approve it we will merge it to master

**Comment by merenkoff** [comment]
@DeRain  Tests are added. So please review it again.

**Comment by DeRain** [comment]
> LGTM if @DeRain will approve it we will merge it to master

This branch is going to be merged into `uns` branch, not in master. We are keeping all uns updates in separate branch since this introduces a big amount of changes and we want to keep master clean while we are working on uns integration.

**Review by DeRain — APPROVED** [review]
👍

**Pull request merged**